datasets/sg/dev_board/crawler.py

```diff
@@ -0,0 +1,71 @@
+import re
+
+from zavod import Context, helpers as h
+from zavod.logic.pep import categorise
+from zavod.shed.zyte_api import fetch_html
+
+
+LINKS = {
+    # Mapping URLs to their respective position names
+    "https://www.edb.gov.sg/en/about-edb/our-team/executive-management.html": "Member of the Executive Committee",
+    "https://www.edb.gov.sg/en/about-edb/our-team/board-members.html": "Board Member",
+    "https://www.edb.gov.sg/en/about-edb/our-team/international-advisory-council.html": "Member of the International Advisory Council",
```

Comment:
I think it's best to include the board name in the position name in this case. I see https://zavod.opensanctions.org/peps/#selecting-a-position-name is a bit vague with "include the organisational body where needed". But I think in this case it's helpful I don't think there are hundreds of boards (https://www.sgdi.gov.sg/statutory-boards) so the number of positions won't get out of hand.

```suggestion
    "https://www.edb.gov.sg/en/about-edb/our-team/executive-management.html": "Member of the Executive Committee of the Economic Development Board",
    "https://www.edb.gov.sg/en/about-edb/our-team/board-members.html": "Board Member of the Economic Development Board",
    "https://www.edb.gov.sg/en/about-edb/our-team/international-advisory-council.html": "Member of the International Advisory Council of the Economic Development Board",
```